### ISSUE TITLE
Penlight failed install

### ISSUE THREAD

**Issue by netyoda** [luarocks/luarocks#1120]
```
sudo luarocks install penlight
Warning: falling back to curl - install luasec to get native HTTPS support
Installing https://luarocks.org/penlight-1.7.0-1.src.rock
Missing dependencies for penlight 1.7.0-1:
   luafilesystem (not installed)

penlight 1.7.0-1 depends on luafilesystem (not installed)
Installing https://luarocks.org/luafilesystem-1.7.0-2.src.rock


Error: LuaRocks 3.0.4 bug (please report at https://github.com/luarocks/luarocks/issues).
Arch.: linux-x86_64
/usr/local/share/lua/5.3/luarocks/manif/writer.lua:418: attempt to index a nil value (local 'package_entry')
stack traceback:
	/usr/local/share/lua/5.3/luarocks/manif/writer.lua:418: in function 'luarocks.manif.writer.remove_from_manifest'
	(...tail calls...)
	/usr/local/share/lua/5.3/luarocks/build.lua:388: in function 'luarocks.build.build_rockspec'
	/usr/local/share/lua/5.3/luarocks/cmd/build.lua:78: in function </usr/local/share/lua/5.3/luarocks/cmd/build.lua:55>
	(...tail calls...)
	/usr/local/share/lua/5.3/luarocks/cmd/build.lua:155: in function 'luarocks.cmd.build.command'
	(...tail calls...)
	/usr/local/share/lua/5.3/luarocks/deps.lua:172: in function 'luarocks.deps.fulfill_dependency'
	/usr/local/share/lua/5.3/luarocks/deps.lua:217: in function 'luarocks.deps.fulfill_dependencies'
	/usr/local/share/lua/5.3/luarocks/build.lua:139: in upvalue 'process_dependencies'
	/usr/local/share/lua/5.3/luarocks/build.lua:379: in function 'luarocks.build.build_rockspec'
	/usr/local/share/lua/5.3/luarocks/cmd/build.lua:78: in function </usr/local/share/lua/5.3/luarocks/cmd/build.lua:55>
	(...tail calls...)
	/usr/local/share/lua/5.3/luarocks/cmd/build.lua:155: in function 'luarocks.cmd.build.command'
	(...tail calls...)
	[C]: in function 'xpcall'
	/usr/local/share/lua/5.3/luarocks/cmd.lua:484: in function 'luarocks.cmd.run_command'
	...cal/lib/luarocks/rocks-5.3/luarocks/3.0.4-1/bin/luarocks:36: in main chunk
	[C]: in ?
```

LFS is already installed

**Comment by daurnimator** [comment]
How did you install luarocks?

**Comment by hishamhm** [comment]
@netyoda Thank you for the report!  This is already fixed in the latest master (fixed in 879c6ef01f94df18a8457a3eb78aeafc4fe0a003, PR #1096). It will be included in the text release.

Closing this as a duplicate of #1095

**Comment by netyoda** [comment]
@hishamhm thank you, rerunning it did get it to work, but I'll update luarocks in when you release the next master.
